framework/src/modules/network/network.js

```diff
@@ -37,26 +37,23 @@ module.exports = class Network {
 
 		this.logger = createLoggerComponent(loggerConfig);
 
-		const initialNodeInfo = await this.channel.invoke('chain:getNodeInfo');
+		const applicationState = await this.channel.invoke(
+			'lisk:getApplicationState'
+		);
 
 		const p2pConfig = {
 			...this.options,
 			nodeInfo: {
-				...initialNodeInfo,
+				...applicationState,
 				wsPort: this.options.nodeInfo.wsPort,
```

Comment:
Do we need this anymore? it's coming from the application state already. Or, there is a chance that it could vary?

Reply:
Application state contains broadhash and height which are constantly being updated by chain module.

Reply:
Oh, I was talking about `wsPort` that why we are explicitly assigning it from `this.options.nodeInfo.wsPort`. We are getting it from `applicationState` anyways so we can remove this line?

Reply:
@jondubois wants to overwrite it for now until config refactoring is done. We have some temporary config in here that overwrites the one used by the application. https://github.com/LiskHQ/lisk/blob/feature/create_network_module/framework/src/modules/network/defaults/index.js

Reply:
Oh yeah so as of now we need it until config refactoring is done